test/apiv2/15-manifest.at

```diff
@@ -8,12 +8,36 @@ id_abc=$(jq -r '.Id' <<<"$output")
 
 t POST /v4.0.0/libpod/manifests/xyz 201 \
     .Id~[0-9a-f]\\{64\\}
-echo xyz $output
 id_xyz=$(jq -r '.Id' <<<"$output")
 
 t GET /v3.4.0/libpod/manifests/$id_abc/exists 204
 t GET /v4.0.0/libpod/manifests/$id_xyz/exists 204
 
+id_abc_image=$($PODMAN_BIN --root $WORKDIR/server_root image build -q --format=docker -<<EOF
+FROM alpine
+RUN >file1
+EOF
+)
+
+id_xyz_image=$($PODMAN_BIN --root $WORKDIR/server_root image build -q --format=docker -<<EOF
+FROM alpine
+RUN >file2
+EOF
+)
+
+t POST /v3.4.0/libpod/manifests/$id_abc/add images="[\"containers-storage:$id_abc_image\"]" 200
+t PUT /v4.0.0/libpod/manifests/$id_xyz operation='update' images="[\"containers-storage:$id_xyz_image\"]" 200
+
+# Run registry container
+# FIXME this fails if python tests have been run first...
+podman run -d --name registry -p 5000:5000 quay.io/libpod/registry:2.7 /entrypoint.sh /etc/docker/registry/config.yml
```

Comment:
There's already a `start_registry()`, could you use it, or document why you can't? At the very least please use `$REGISTRY_IMAGE` instead of hardcoding.

Also, please add a cleanup.

Also, the `FIXME` concerns me: we make no guarantees about test order. Can you look into it and, if possible, resolve the problem?

Reply:
Thanks for pointing out to `start_registry`.

The line running the registry along with `FIXME` comment was copied from `12-imagesMore.at` file

Reply:
> Also, please add a cleanup.

`podman rmi -a -f` is already there

Reply:
That's sort of a sledgehammer. It spits out scary warnings, and it also adds ten unnecessary seconds to a test run (every second makes a difference when you run hundreds of CI jobs each day). A friendlier and more readable cleanup might be:
```bash
# Done. Stop registry, and clean up stray images
podman rm -f -t 0 registry
podman rmi -a
```
(no `-f` needed in `rmi` because there are no containers; and if there are, the rmi will fail, and that will be good because it will indicate something went wrong)

Reply:
@edsantiago I put `start_service` in both `12-imageMore.at` and `15-manifest.at`

Reply:
Also added `cleanUpManifestTest`

Reply:
> podman rm -f -t 0 registry

It's not needed if using `start_registry` because there's `stop_registry` at the end of entire test suite.